test/browser/support/TestHarness.js

```diff
@@ -91,7 +91,7 @@ module.exports = class TestHarness {
     return keyList;
   }
 
-  async validateTxMAEventData(keyList) {
+  async validateTxMAEventData(keyList, journeyType) {
```

Comment:
What do you think about passing in the schemaName rather than the journeyType? Just saves the if/else

Reply:
Yeah I can definitely look into that as an improvement, would it make sense to merge this now to unblock the pipeline while we work on this improvement?

Reply:
Ok sure